include/deal.II/base/array_view.h

```diff
@@ -1134,6 +1134,21 @@ make_array_view(const Table<2, ElementType> &                   table,
 
 
 
+/*
+ * Create a view that doesn't modify the container it points to.
```

Comment:
actually, I think what you want to say is that it doesn't allow the container *to be modified*

Reply:
That's probably a better wording indeed.